### PR TITLE
@damassi => Better styling for contents of follow artist popover

### PR DESCRIPTION
**Pull request by mzikherman**
screen shot 2018-11-14 at 12 26 33 pm

Looks like ^ , starting to come together!

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 2d57cdef6d9bdb6aa10f270a03e47c2d16c54066

https://deploy-preview-1545--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit f4ccd5a5266999280868aaa5d26091ecf44b0ddc

https://deploy-preview-1545--artsy-reaction.netlify.com

File: src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx [hunk 1]
```diff
@@ -59,15 +60,45 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
...
     const imageUrl = get(artist, a => a.image.cropped.url)
+    const { _id: artistID } = artist
     return (
-      <div onClick={() => this.handleClick(artist._id)}>
-        <span>{artist.name}</span>
-        <Image src={imageUrl} />
-      </div>
+      <>
+        <Flex mb={2} mt={2}>
+          <Image mr={1} style={{ borderRadius: "17px" }} src={imageUrl} />
```
Comment (damassi) [inline src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx]: No custom styles -- should use `<Avatar />`

Comment (mzikherman) [reply src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx]: oo that's perfect! But the design for this has the image at 34px which is not supported: https://github.com/artsy/palette/blob/2a799c19b073b231f6246ebda790f4a78d8dca11/src/elements/Avatar/Avatar.tsx#L20

Can that be overridden/extended in this specific use case?

Comment (damassi) [reply src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx]: I think the 34px ask needs to be be pushed back to Jeffery with a link to the three available sizes.

Comment (mzikherman) [reply src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx]: Cool, I'll go with the smallest one for now, and ask about it in the JIRA ticket.

File: src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx [hunk 2]
```diff
@@ -59,15 +60,47 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
...
+    const { _id: artistID } = artist
     return (
-      <div onClick={() => this.handleClick(artist._id)}>
-        <span>{artist.name}</span>
-        <Image src={imageUrl} />
-      </div>
+      <>
+        <Flex my={2}>
+          <Box mr={1}>
+            <Avatar size="xs" src={imageUrl} />
```
Comment (damassi) [inline src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx]: much nicer 😄

**Comment by mzikherman** [comment]
k, merging!

**Pull request merged**